game/src/engine/physics/PhysicsService.cpp

```diff
@@ -535,6 +547,15 @@ const PxVec3& PhysicsService::GetGravity() const
     return kGravity;
 }
 
+bool PhysicsService::GetDisplayPauseBoolean()
+{
+    return display_pause_;
+}
+
+void PhysicsService::SetDisplayPauseBoolean(bool boolean){
+    display_pause_ = boolean;
+}
```

Comment:
Can you rename this to `SetPaused` and an equivalent `GetPaused` or something along those lines instead?

I just find that usually adding type info into func/var names doesn't end up being too useful in the long run, since intellisense/autocomplete will tell you what the types are when you hover over the func/var. Also, this doesn't really apply to this case, but if we were to ever change the type of this var, then all the func names would _also_ have to change, which can be cumbersome if it's a func used everywhere (for example, if we had a `GetPositionVec3()` func and we changed pos to be a `vec4` instead, now we have to go do some extra work and _also_ rename every use of that func so that it's not misleading). Funcs returning booleans are (from what I've seen in other code at least) usually an exception, but most commonly they're named things like `IsSomething()` or `SetIsSomething()` as opposed to `SetSomethingBool()` `GetSomethingBool()`